assets/promptflow/models/rai-eval-ui-dag-flow/description.md

```diff
@@ -0,0 +1,42 @@
+The Q&A quality and safety evaluation flow will evaluate the Q&A systems by leveraging the state-of-the-art Large Language Models (LLM) to measure the quality and safety of your responses. Utilizing GPT and GPT embedding model to assist with measurements aims to achieve a high agreement with human evaluations compared to traditional mathematical measurements.
+
+
+### Inference samples
+
+Inference type|CLI|VS Code Extension
+|--|--|--|
+Real time|<a href="https://microsoft.github.io/promptflow/how-to-guides/deploy-a-flow/index.html" target="_blank">deploy-promptflow-model-cli-example</a>|<a href="https://microsoft.github.io/promptflow/how-to-guides/deploy-a-flow/index.html" target="_blank">deploy-promptflow-model-vscode-extension-example</a>
+Batch | N/A | N/A
+
+### Sample inputs and outputs (for real-time inference)
+
+#### Sample input
+```json
+{
+    "inputs": {
```

Comment:
Does the sample input listed here consist with the current schema?

Reply:
synced with Qun offline. since the flow doesn't show up in the promptflow gallery, it's okay for now.